### PR TITLE
CFP

### PR DESCRIPTION
**Pull request by zkriszti**

### CODE REVIEW

**Comment by greg-nagy** [comment]
ty

File: ui/cfp-button.tsx [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
+}
+
+function CfpButton({url, children}: CfpButtonProps) {
     return (
         <div className={styles.main}>
-            <Link href="/" className={styles.button}>
-                Call for Papers
-            </Link>
-        </div>);
+          <button className={styles.button}>
```
Comment (necccc) [inline ui/cfp-button.tsx:14]: Could you style the next Link component as a button?

Wrapping links with <button> components is not a good practice (buttons are form/logical action elements, which may not suppose to navigate away from the page)

**Review by necccc — CHANGES_REQUESTED** [review]

**Comment by necccc** [comment]
Hey @zkriszti  added you as a collaborator, so you can open PR-s from project branches - which will enable netlify to generate previews

File: ui/cfp-button.tsx (hunk 1 above)
Comment (greg-nagy) [reply ui/cfp-button.tsx:14]: that's a good catch. I will fix it, thx

Comment (greg-nagy) [reply ui/cfp-button.tsx:14]: ff2779049b6130c02d6c8d6e0c8a4ab7a24e51e6

**Pull request merged**